### PR TITLE
BVAL-604 Avoiding forced line wraps

### PR DESCRIPTION
**Pull request by gunnarmorling**
https://hibernate.atlassian.net/browse/BVAL-604

### CODE REVIEW

File: src/main/java/javax/validation/ConstraintValidatorContext.java [hunk 1]
```diff
@@ -131,14 +131,16 @@
 	 * Cross-parameter constraints on a method can create a node specific
 	 * to a particular parameter if required. Let's explore a few examples:
 	 * <pre>
-	 * //Cross-parameter constraint on method createUser(String password, String passwordRepeat)
+	 * //Cross-parameter constraint on method
+	 * createUser(String password, String passwordRepeat)
```
Comment (gsmet) [inline src/main/java/javax/validation/ConstraintValidatorContext.java]: Missing `//`?

File: src/main/java/javax/validation/ConstraintValidatorContext.java [hunk 2]
```diff
@@ -326,7 +330,8 @@ ContainerElementNodeBuilderCustomizableContext addContainerElementNode(String na
 			 *
 			 * @since 2.0
 			 */
-			LeafNodeBuilderCustomizableContext inContainer(Class<?> containerClass, Integer typeArgumentIndex);
+			LeafNodeBuilderCustomizableContext inContainer(Class<?> containerClass,
+														   Integer typeArgumentIndex);
```
Comment (gsmet) [inline src/main/java/javax/validation/ConstraintValidatorContext.java]: Do we really want to index like that? It's a pain to read in GitHub's diff.

Comment (gunnarmorling) [inline src/main/java/javax/validation/ConstraintValidatorContext.java]: I agree with you and don't like this style either. But it's what's used in the API in other places AFAICS, so I'd prefer to stick to it as far as possible.

File: src/main/java/javax/validation/ConstraintValidatorContext.java (hunk 1 above)
Comment (gunnarmorling) [inline src/main/java/javax/validation/ConstraintValidatorContext.java]: Code catch - reworked and force-pushed.

**Pull request merged**